src/router.ts

```diff
@@ -980,7 +980,7 @@ export async function handleClearedVoter(
   userInfo.lastVoterMessageSecsFromEpoch = nowSecondsEpoch;
 
   await SlackApiUtil.sendMessage(
-    `*${userId.substring(0, 5)}:* ${userOptions.userMessage}`,
+    `${userOptions.userMessage}`,
```

Comment:
This is only one of the points at which messages from the voter are relayed to Slack, so you'll have to remove the prefix for all, if that's what you intended. See line 916 for example.

Reply:
> One high-level thought: is there a way we can make clear that the (singular) messages posted by the bot are coming from the voter? That's what the userId prefix was intended to do, but I can see an argument that this quickly becomes clear to a new volunteer. This might be a question for Brenna/Fuzzy, but maybe we should rename the bot (which we can do in the app settings), or change its avatar? The other thing to consider is that the bot does more than just relay voter messages -- it also posts when a voter status or volunteer is updated, or when a voter is routed. So we need to make sure those are differentiated from voter messages.

Yeah.  The bot-related messages still have the bold **Helpline** prefix on them... I was hoping that would be sufficiently clear (since voter's can bold their stuff)?  At the end of the day the bot only has one avatar, so we have to share somehow.

Reply:
Sounds good to me (I think you mean the **Operator:** prefix, right?). I think it'll be clear to volunteers what's going on, so this sounds good.